Sources/Workaround-SPM/File.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  File.swift
+//  
+//
+//  Created by BJ Homer on 7/27/21.
+//
+
```

Comment:
What do you think of replacing this with a few lines explaining why we need this workaround here?

Reply:
We're not likely to ship Xcode 13 CI support until about the time macOS Monterey ships (typically most of the bugs are worked out by then). That may change if there are critical features we need from XC13.